contracts/StablecoinConverter.sol

```diff
@@ -165,10 +165,10 @@ contract StablecoinConverter is EpochTokenLocker {
 
     function submitSolution(
         uint32 batchIndex,
-        address[] memory owners,  //tradeData is submitted as arrays
+        address[] memory owners,          // tradeData is submitted as arrays
```

Comment:
Not sure how long this alignment will survive.

Reply:
Ya, sad... but understandable.